src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3792,6 +3792,45 @@ void SpellMgr::LoadSpellInfoCorrections()
         //! HACK: This spell break quest complete for alliance and on retail not used
         spellInfo->Effects[EFFECT_0].Effect = 0;
     });
+	
+	 //
```

Comment:
Tabs

Reply:
@Wilcalaf : Depending on the editor you use, look up how to configure it to automatically change inserted tabs to 4 spaces.

Reply:
@tkrokli Aye i fixed it already this file was edited before your reviews - thanks :)